### ISSUE TITLE
 Accessibility

### ISSUE THREAD

**Issue by rossjanderson** [arXiv/html_feedback#497]
### Description

Great idea, but the text is displayed in the right-hand half of the screen. For accessibility and indeed legibility, can we please have single column? It's a complete waste to give half the page to the section heading. Who cares about the headings? Put them inline, or just throw them away. For a first rapid reading, they don't matter.By the way, this box itself sucks. Always test your stuff in a browser where the text has been upsized 3 notches. Not everyone is 22 years old with 2020 vision!

### (Optional:) Please add any files, screenshots, or other information here.

Upsize the text in your browser by three notches, then test your product

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

fd192f49-4e82-442f-8ccc-e5d8b43d0345

### Paper URL

https://arxiv.org/html/2312.14202v1

### Browser

Firefox/122.0

### Device Type

Desktop

**Comment by dginev** [comment]
> Great idea, but the text is displayed in the right-hand half of the screen.

Could you share a screenshot? The article renders normally for me in Firefox.

> Who cares about the headings?

Most readers. I am pretty confident you are a marginalized minority on the question. But there is a positive part of my answer - HTML is very easy to theme with custom CSS. I am still hoping that we will see unofficial browser extensions that re-skin the arXiv article web view, and eventually - with enough interest - maybe even have arXiv-provided themes to choose from.

**Comment by rossjanderson** [comment]
Upsize the text three notches!

Regards

Ross

**Comment by rossjanderson** [comment]
OK, so I go to one of my own papers

 https://arxiv.org/html/2303.14178v3

using firefox 122.0.1 on a 2019 macbook pro with 1.7GHz Intel chip Intel
running Iris Plus Graphics 645 1536 MB. I see a "failed: parcolumns". The
appearance is the attached screenshot.jpg.

How do I make the left-hand column go away and get single-column text?

Regards

Ross

**Comment by dginev** [comment]
Hi again. The attachment didn't come through, but looking at your paper I can guess you mean the table-of-contents sidebar. You can click on the left-pointing arrow at the top, which will collapse the entire sidebar away, leaving only the content. For reference:

<img src="https://github.com/arXiv/html_feedback/assets/348975/1412405f-6835-40b8-b645-4c473b7157c2" width="200px">


<br>
Btw: In Firefox, you can also use "reader mode" which will help declutter the page from any auxiliary accessories.
https://support.mozilla.org/en-US/kb/firefox-reader-view-clutter-free-web-pages

**Comment by dginev** [comment]
Ah, but here I have a usability comment for @chrisjcameron . When the page is zoomed in to a very high level (see my 300% screenshot above) the header grows to a level where it completely hides the arrow for hiding the sidebar. Ideally the header should push the sidebar down (or if that is too tricky - the z-index of the sidebar should be higher than the header, so that it stays on top).

**Comment by rossjanderson** [comment]
OK, I understand now. Because I upsize the text, I can't see the arrow
button to get rid of the sidebar, which as a result takes up half the
screen real estate, making accessibility worse.

Solution 1: no sidebar. This would be my preferred option. Academic papers
don't have sidebars; they have section headings, inline in the text. A
sidebar  is entirely wrong and inappropriate in the context of a research
paper.

Solution 2: make the stop-faffng-about button so enormous so that it is
never ever dominated by the header.

Soliution 3: get rid of the header, or at least make it trivial to click it
away.

Solution 4: don't have both a sidebar and a header. Put the header stuff at
the top of the sidebar, and have a really big obvious button to click to
get rid of the lot.

Solution 5: if someone upsizes the text, just ditch all the fancy stuff
like headers and sidebars and display text only.

What's your preference?

Regards

Ross

**Comment by chrisjcameron** [comment]
I appreciate the discussion here and will raise it with the UI team for consideration.

**Comment by dginev** [comment]
The page scaffold for HTML papers has been refreshed for new articles starting March 2026. 
The article reported in this issue (thank you!) has been updated to include that scaffold.

A full update of all arXiv HTML articles will follow at a later date.

You can use the new header buttons to hide/show the navigational menu, as well as the header itself. That choice will be saved for future visits in the same browser.
